autowarefoundation/autoware
# Add missing install for image resources on camera calibrator

## Status
**PRODUCTION / DEVELOPMENT**

## Description
Fixes autowarefoundation/autoware_ai#626 

## Related PRs
autowarefoundation/autoware#2000 autowarefoundation/autoware#1990 autowarefoundation/autoware#1977 


## Steps to Test or Reproduce

1. In a sourced terminal
`rosrun autoware_camera_lidar_calibrator cameracalibrator.py --square 0.1 --size "8x6" image:=/image_raw`
1. Play rosbag or connect camera, publishing `sensor_msgs/Image` to `/image_raw`
The Calibrator UI should load.
